docs/tools/debugging/proposer-based-timestamps-runbook.md

```diff
@@ -0,0 +1,207 @@
+If you are running a Prometheus collector, navigate to the query web interface and select the 'Graph' tab.
+
+Issue a query for the following:
+
+```
+sum(tendermint_consensus_quorum_prevote_delay) by (proposer_address)
+```
+This query will graph the difference in seconds for each proposer on the network.
+
+If the value is much larger for some proposers, then the issue is likely related to the clock
+synchronization of their nodes. Contact those proposers and ensure that their nodes
+are properly connected to NTP using the steps for [debugging a single node](#debugging-a-single-node).
+
+If the value is relatively similar for all proposers you should next compare this
+value to the `SynchronyParams` values for the network. Continue to the [Checking
+Sychrony](checking-synchrony) steps.
+
+#### From the `/metrics` url
+
+If you are not running a Prometheus collector, navigate to the `/metrics` endpoint
+exposed on the Prometheus metrics port.
+
+Search for the `tendermint_consensus_quorum_prevote_delay` metric. There will be one
+entry of this metric for each `proposer_address`. If the value of this metric is
+much larger for some proposers, then the issue is likely related to synchronization of their
+nodes with NTP. Contact those proposers and ensure that their nodes are properly connected
+to NTP using the steps for [debugging a single node](#debugging-a-single-node).
+
+If the values are relatively similar for all proposers you should next compare,
+you'll need to compare this value to the `SynchronyParams` for the network. Continue
+to the [Checking Sychrony](checking-synchrony) steps.
+
+### Checking Synchrony
+
+To determine the currently configured `SynchronyParams` for your network, issue a
+request to your node's RPC endpoint. For a node running locally with the RPC server
+exposed on port `26657`, run the following command:
+
+```
+# curl localhost:26657/consensus_params
+```
+
+The json output will contain a field named `synchrony`, with the following structure:
+
+```json
+{
+  "precision": "500000000",
+  "message_delay": "3000000000"
+}
+```
+
+The `precision` and `message_delay` values returned are listed in nanoseconds.
+Remember, `tendermint_consensus_quorum_prevote_delay` is listed in seconds.
+If the `tendermint_consensus_quorum_prevote_delay` value approaches the sum of `precision` and `message_delay`,
+then the value selected for these parameters is too small. Your application will
+need to be modified update the `SynchronyParams` to have larger values.
+
+### Updating SynchronyParams
+
+The `SynchronyParams` are `ConsensusParameters` which means they are set and updated
+by the application running alongside Tendermint. Updates to these parameters must
+be passed to the application during the `FinalizeBlock` ABCI method call.
+
+If the application was built using the CosmosSDK, then these parameters can be updated
+programatically using a governance proposal. For more information, see the [CosmosSDK
+documentation]().
```

Comment:
My searching has yet to turn up this documentation.

Reply:
To clarify, the documentation I'm hoping to link here will be just on what a governance proposal is, not necessarily the specific CosmosSDK documentation on PBTS.